.gitOriginal/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/master
```

Comment:
I have no idea what this is, but .gitOriginal should not be commited. Same to .vscode files